packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md

```diff
@@ -1,9 +1,9 @@
 The `OktaAuthGuard` can protect a route to a single component as well with `canActivate` property.
 
-1. Add the `canActivate` property to the `routes` array in `app-routing.module.ts`:
+1. Add the `canActivate` property to the `routes` array in `app.routes.ts`:
 
    ```ts
    { path: 'profile', component: ProfileComponent, canActivate: [OktaAuthGuard] }
    ```
 
-The single `/profile` route is protected directly. If you're not authenticated and go to this page, you must sign in before being routed to the `ProfileComponent`.
+The single `/profile` route is protected directly. If you're not authenticated and go to this page, you must click sign in before being routed to the `ProfileComponent`.
```

Comment:
Should sign in be capitalized or formatted as a button?

Reply:
Yes, this is a reference to a sign in button. I can add bold formating for it.